tests/publiccloud/transfer_repos.pm

```diff
@@ -50,9 +50,9 @@ sub run {
         s/https?:\/\/// for @repos;
 
         # Create list of directories for rsync
-        my $directories;
-        $directories .= "$_\n" foreach @repos;
```

Comment:
Just a note: changing the line to `$directories .= "$_\\n" foreach @repos;` (extra backslash) would have worked as well.

Reply:
> Just a note: changing the line to `$directories .= "$_\\n" foreach @repos;` (extra backslash) would have worked as well.

Been there, done that.  Didn't work :)